app-office/sc-im/sc-im-0.8.2.ebuild

```diff
@@ -0,0 +1,143 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-1 luajit )
+
+inherit lua-single toolchain-funcs flag-o-matic
+
+DESCRIPTION="Spreadsheet Calculator Improvised -- An ncurses spreadsheet program for terminal"
+HOMEPAGE="https://github.com/andmarti1424/sc-im"
+SRC_URI="https://github.com/andmarti1424/sc-im/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-4"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="X plots wayland xls xlsx lua ods tmux"
+REQUIRED_USE="lua? ( ${LUA_REQUIRED_USE} )"
+
+DEPEND="
+	sys-libs/ncurses
+
+	lua? (
+		${LUA_DEPS}
+	)
+	ods? (
+		dev-libs/libxml2
+		dev-libs/libzip
+	)
+	plots? ( sci-visualization/gnuplot )
+	tmux? ( app-misc/tmux )
+	wayland? ( gui-apps/wl-clipboard )
+	X? ( x11-misc/xclip )
+	xls? (
+		dev-libs/libxls
+	)
+	xlsx? (
+		dev-libs/libxlsxwriter
+		dev-libs/libxml2
+		dev-libs/libzip
+	)
+"
+RDEPEND="${DEPEND}"
+BDEPEND="virtual/pkgconfig"
+S="${WORKDIR}/${P}/src"
```

Comment:
I usually prefer to put this directly under SRC_URI (no blank new line between) because of how closely related it is to SRC_URI